### PR TITLE
Fix minor typos encountered while developing the python client

### PR DESCRIPTION
**Pull request by louisguitton**
Part 2 of my user feedback (after my email to @dissroff ) are a couple of minor typos I found while developing https://github.com/Onefootball/onefootball-network-api-py

- [x] leave comments to put them up for discussion
- [ ] add them in other languages

### CODE REVIEW

File: source/api.rst [hunk 1]
```diff
@@ -187,10 +187,11 @@ Publishing an article
           "published": "2010-01-02T15:04:05Z",
           "modified": "2010-01-02T15:04:05Z",
           "content": "Article content",
-          "title": "Article title"
+          "title": "Article title",
           "image_url": "https://your-blog.com/images/1.png",
           "image_width": 200,
           "image_height": 100,
+          "breaking_news": false,
```
Comment (louisguitton) [inline source/api.rst:194]: also missing for curl and golang. If you want it in, I can add it for all languages

File: source/api.rst [hunk 2]
```diff
@@ -247,18 +248,20 @@ Publishing an article
...
 | ``source_url``     | required   | The link to the article as published on your website. Place your homepage url if the article is not published anywhere else.                                                                                                                  |
 +--------------------+------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``language``       | required   | The language of the article. Valid choices are ``en``, ``de``, ``es``, ``fr``, ``br``, and ``it``.                                                                                                                                            |
+| ``language``       | required   | The language of the article. Valid choices are ``en``, ``de``, ``es``, ``fr``, ``br``, ``it``, ``id``, ``ko``, ``pt`` and ``ru``.                                                                                                             |
```
Comment (louisguitton) [inline source/api.rst:251]: Found out that the Enum doesn't match the Enum in here https://swaggerui.onefootball.com/?url=/projects/creators-network/docs/openapi.yml#/Posts%20API/post_v1_posts 
image

File: source/api.rst [hunk 3]
```diff
@@ -247,18 +248,20 @@ Publishing an article
...
 | ``published``      | required   | The time that the article was published. If in doubt, use the current time.                                                                                                                                                                   |
 +--------------------+------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``modified``       | required   | The time that the article was last updated. If in doubt, use the current time.                                                                                                                                                                |
+| ``modified``       | optional   | The time that the article was last updated. If in doubt, use the current time.                                                                                                                                                                |
```
Comment (louisguitton) [inline source/api.rst:255]: found out that this was not required

File: source/api.rst [hunk 4]
```diff
@@ -247,18 +248,20 @@ Publishing an article
...
 +--------------------+------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | ``image_url``      | optional   | An optional field for the URL of the article's featured image. If provided, ``image_width`` and ``image_height`` should be provided as well (the image's width and height in pixels).                                                         |
 +--------------------+------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``breaking_news``  | optional   |                                                                                                                                                                                                                                               |
```
Comment (louisguitton) [inline source/api.rst:263]: I'll you add a description for this

**Review by automaticalldramatic — APPROVED** [review]

File: source/api.rst (hunk 2 above)
Comment (automaticalldramatic) [reply source/api.rst:251]: We might be adding nl to this but I guess that's a task not for now

**Review by automaticalldramatic — APPROVED** [review]

File: source/api.rst (hunk 4 above)
Comment (automaticalldramatic) [reply source/api.rst:263]: i guess this is for dani to add the details - i am approving it but wont merge until @dissroff confirms this

File: source/api.rst (hunk 1 above)
Comment (dissroff) [reply source/api.rst:194]: No, we don't want to tell people this as that feature does not do anything yet And we don't want to confuse people.

File: source/api.rst (hunk 2 above)
Comment (dissroff) [reply source/api.rst:251]: No, we don't want to tell people this as it is not applicable yet And we don't want to confuse people.

File: source/api.rst (hunk 3 above)
Comment (dissroff) [reply source/api.rst:255]: No, we don't want to tell people this as we don't want to confuse people.

File: source/api.rst (hunk 4 above)
Comment (dissroff) [reply source/api.rst:263]: No, we don't want to tell people this as This feature doesn't do anything yet we don't want to confuse people.

**Pull request closed without merging**